src/Events/Editor/Full_Site/Templates.php

```diff
@@ -82,9 +87,71 @@ public function get_template_events_archive() {
 		$template->content        = Template_Utils::inject_theme_attribute_in_content( $template_content );
 		$template->slug           = static::$archive_slug;
 		$template->source         = 'custom';
+		$template->type           = 'wp_template';
+		$template->title          = esc_html__( 'Calendar Views (Event Archive)', 'the-events-calendar' );
```

Comment:
```suggestion
		$template->title          = esc_html_x( 'Calendar Views (Event Archive)', 'The Full Site editor block navigation title' 'the-events-calendar' );
```
Perhaps something like this? I think we want to use context variations most of the time now right?

Reply:
Updates